native/spark-expr/src/agg_funcs/stddev.rs

```diff
@@ -56,11 +57,11 @@ impl Stddev {
         Self {
             name: name.into(),
             signature: Signature::coercible(
-                vec![
+                vec![Coercion::new_exact(
```

Comment:
What does this `Coercion` do?

Reply:
This use of `Coercion::new_exact` means that we only accept the specific type (`Float64`) and will not perform any coercion.

From DataFusion docs:

```
/// There are two variants:
///
/// * `Exact` - Only accepts arguments that exactly match the desired type
/// * `Implicit` - Accepts the desired type and can coerce from specified source types
```